package.json

```
@@ -17,8 +17,9 @@
     "start_new": "indexName=npm-search-new bootstrapIndexName=npm-search-new.tmp UV_THREADPOOL_SIZE=64 node --max-old-space-size=1500 dist/index.js",
     "test:watch": "jest --watchAll --no-watchman",
     "test": "jest --forceExit",
-    "docker:build": "./scripts/build.sh",
-    "docker:release": "VERSION=$(node -e \"console.log(require('./package.json').version)\") && echo \"Releasing $VERSION\" && docker push algolia/npm-search && docker push algolia/npm-search:$VERSION"
+    "publish:check": "node ./scripts/publish-check.js",
+    "publish:github": "./scripts/publish-github",
```

this could just be:

```suggestion
    "publish:github": "semantic-release",
```

and the file removed

---

Right. I initially had the external script because there was a bit more code before and after, but you're right that it can now be simplified